src/deadline/nuke_submitter/deadline_submitter_for_nuke.py

```diff
@@ -144,6 +172,12 @@ def _get_job_template(settings: RenderSubmitterUISettings) -> dict[str, Any]:
     # Set the View parameter allowed values
     parameter_def_map["View"]["allowedValues"] = ["All Views"] + sorted(nuke.views())
 
+    # if OCIO is disabled, remove OCIO path from the template
+    if nuke_ocio.is_OCIO_enabled():
+        _add_ocio_path_to_job_template(job_template)
+    else:
+        _remove_ocio_path_from_job_template(job_template)
```

Comment:
What's the use case for removing it? Is the OCIO path already there in the template, I thought it was only added if OCIO was enabled two lines above. Can the param only be added if OCIO is being used and the default left the same?

Reply:
The remove method is to remove any OCIO information from the job_template's parameter definitions. If there is, I think it will complain about not having the default value, etc. Also, I thought if its not being used, it is ideal to remove it from the parameter definition. Please let me know how you think about it 🙏 

The upper part of adding is to add the `jobEnvironment` definition for it using the parameter definition that is in default job template. Please let me know if it makes sense!

Reply:
Can we remove the parameter entirely from the default job template though? Only add the parameter into the template if OCIO is enabled. Then the remove method wouldn't be needed at all and OCIO relevant stuff would only be included in enabled

Reply:
Spoke offline, personally I'm of the opinion that the parameter without the environment and vice versa is not really useful so I don't see why we would bake one into the default template, but not the other. Given Gizmo handling already follows this pattern though I'm not going to block the review on it, but leave it up to the owners to decide whether both OCIO and Gizmo parameters should be in the default template or not.